packages/nextjs/components/project/ProjectHeader.tsx

```diff
@@ -24,14 +24,14 @@ const ProjectHeader = ({ project }: { project: Project }) => {
   const { state } = useBallot();
   const [isDropdownOpen, setIsDropdownOpen] = useState(false);
   const dropdownRef = useRef<HTMLDivElement | null>(null);
-  const [isAllocationError, setIsAllocationError] = useState(false);
+  // const [isAllocationError, setIsAllocationError] = useState(false);
   const [prevProjectAllocation] = state.projects.filter(item => item._id === project._id);
-  const [newAllocation, setNewAllocation] = useState<number>(prevProjectAllocation?.allocation ?? 0);
+  const [newAllocation /*setNewAllocation */] = useState<number>(prevProjectAllocation?.allocation ?? 0);
 
-  const [loadingMessage, setLoadingMessage] = useState("");
-  const [successMessage, setSuccessMessage] = useState("");
-  const [isLoading, setIsLoading] = useState(false);
-  const [isSuccess, setIsSuccess] = useState(false);
+  // const [loadingMessage, setLoadingMessage] = useState("");
+  // const [successMessage, setSuccessMessage] = useState("");
+  // const [isLoading, setIsLoading] = useState(false);
+  // const [isSuccess, setIsSuccess] = useState(false);
```

Comment:
This component is currently missing the ability to add a project to the ballot. I am overlooking these commented out lines thinking that they may be related. We should make an issue for restoring this functionality but it is not a problem with your PR specifically.

Reply:
I will work on it and make sure the button is working